testplan/common/utils/logger.py

```diff
@@ -82,7 +82,12 @@ def driver_info(self, msg, *args, **kwargs):
 
     def log_test_status(self, name, passed, indent=0, level=TEST_INFO):
         """Shortcut to log a pass/fail status for a test."""
-        pass_label = Color.green('Pass') if passed else Color.red('Fail')
+        if passed is True:
```

Comment:
Just FYI for anyone else wondering "Shouldn't it be `if passed:` or `if passed == True`?": As of Python 2.3 `True` and `False` are singletons. [link](https://stackoverflow.com/q/27276610)

Reply:
We have this pattern in a number of places. It's ok when there is a good reason you for distinguising between `True` and other truthy values, but the times you actually need to do that should be quite rare (and a reason given as a comment). Normally `if passed:` is more readable and conventional.